tests/launcher.py

```diff
@@ -538,6 +540,9 @@ def launch(self, instance: nodeDefinition):
 
     def start_all(self):
         unshare(CLONE_NEWNET)
+        for index, name in socket.if_nameindex():
+            if getInterfaceFlags(name) & IFF_LOOPBACK:
+                os.system(f'ip link set {name} up')
```

Comment:
This adds a new dependency to run the tests. Need to update the documentation or (preferably) just do the ioctl here to up the interface.

Reply:
The `iproute` or `iproute2` package containing `ip` has been a dependency of `ubuntu-minimal` for over a decade and Ubuntu doesn't run without it.  It's not a dependency that needs to be explicit in application packages.

Reply:
If the code in this PR was actually used you'd see it won't even work in CI right now since `iproute2` is _not_ included in ubuntu's official docker image, nor is it indirectly included after installing 300+ dep packages we use for building.

ex, ubuntu22 image used on main:
```
$ docker run --rm -it ghcr.io/antelopeio/builders:75cd0fb7364b148e5d0a663ca8f4aed434571489a69dd228628732f8a8ef2a49
Unable to find image 'ghcr.io/antelopeio/builders:75cd0fb7364b148e5d0a663ca8f4aed434571489a69dd228628732f8a8ef2a49' locally
75cd0fb7364b148e5d0a663ca8f4aed434571489a69dd228628732f8a8ef2a49: Pulling from antelopeio/builders
e96e057aae67: Pull complete
faf78f8066b7: Pull complete
Digest: sha256:fa291238c1cb85d37c6f8c374de55e05a1766332f0af182204cb1b7007b05809
Status: Downloaded newer image for ghcr.io/antelopeio/builders:75cd0fb7364b148e5d0a663ca8f4aed434571489a69dd228628732f8a8ef2a49
root@a2f548020b32:/# ip
bash: ip: command not found
```

But really you should just do the ioctl yourself instead of adding a new dependency since it's straightforward to do the ioctl.

Reply:
And if you try `/sbin/ip`?

Reply:
Unbelievable.  It doesn't.

Whatever.  I already pushed a change to directly calling ioctl.

Reply:
```
$ docker run --rm -it ghcr.io/antelopeio/builders:75cd0fb7364b148e5d0a663ca8f4aed434571489a69dd228628732f8a8ef2a49
root@ca9038f4d6ea:/# /sbin/ip
bash: /sbin/ip: No such file or directory
```